src/app/components/core/manage-services/list-manage-services.component.ts

```diff
@@ -0,0 +1,342 @@
+        }
+      }
+    );
+  }
+
+  /**
+    * Open Settings modal
+    */
+   openServiceModal(service) {
+    this.serviceModal.toggleModal(true);
+    this.serviceModal.getServiceInfo(service, this.availableServicePkgs);
+  }
+
+  onNotify() {
+    // added 3 second wait after redirecting list page from modal beacuse it takes sometime to get data from API
+    setTimeout(() => {
+      this.showServices();
+    }, 3000);
+  }
+
+  deleteService() {
+    this.ngProgress.start();
+    this.servicesApiService.deleteService(this.service.name).subscribe(
+      (data: any) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        this.alertService.success(data["result"], true);
+        this.closeModal("delete-confirmation-dialog");
+        this.onNotify();
+      },
+      (error) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        if (error.status === 0) {
+          console.log("service down ", error);
+        } else {
+          this.alertService.error(error.statusText);
+        }
+      }
+    );
+  }
+
+  enableService() {
+    this.ngProgress.start();
+    this.schedulesService.enableScheduleByName(this.service.name).subscribe(
+      (data) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        this.alertService.success(data["message"], true);
+        this.closeModal('confirmation-dialog');
+        this.onNotify();
+      },
+      (error) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        if (error.status === 0) {
+          console.log("service down ", error);
+        } else {
+          this.alertService.error(error.statusText);
+        }
+      }
+    );
+  }
+
+  disableService() {
+    this.ngProgress.start();
+    this.schedulesService.disableScheduleByName(this.service.name).subscribe(
+      (data) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        this.alertService.success(data["message"], true);
+        this.closeModal('confirmation-dialog');
+        this.onNotify();
+      },
+      (error) => {
+        this.ngProgress.done();
+        this.reenableButton.emit(false);
+        if (error.status === 0) {
+          console.log("service down ", error);
+        } else {
+          this.alertService.error(error.statusText);
+        }
+      }
+    );
+  }
+
+  applyClass(serviceStatus: string) {
+    if (serviceStatus.toLowerCase() === "running") {
+      return "is-success";
+    }
+    if (serviceStatus.toLowerCase() === "shutdown") {
+      return "is-light";
+    }
+    if (serviceStatus.toLowerCase() === "unresponsive") {
+      return "is-warning";
+    }
+    if (serviceStatus.toLowerCase() === "failed") {
+      return "is-danger";
+    }
+  }
+
+  setService(service) {
+    this.service = service;
+  }
+
+  stateUpdate() {
+    if (["shutdown", "disabled"].includes(this.service.state) === true) {
```

Comment:
if (["shutdown", "disabled"].includes(this.service.state)) it will work the same.